extensions/wikia/BannerNotifications/css/BannerNotifications.monobook.scss

```diff
@@ -0,0 +1,6 @@
+@import 'skins/shared/styles/buttons/buttons-wikia-chiclet';
```

Comment:
It's here so the 'close' button looks like it does on Oasis. Deleting it would make the close button look unstyled. I could add some minimal styles so it's visible enough, I just added these styles to make BN on both skins look the similar. WHat do you think, @lizlux ?